clients/logger/logger.go

```diff
@@ -71,6 +71,28 @@ func NewClient(owningServiceName string, isRemote bool, logTarget string, logLev
 		logLevel = models.InfoLog
 	}
 
+	lc := newClient(owningServiceName, isRemote, logTarget, logLevel)
+
+	if logTarget == "" {
+		lc.Error("logTarget cannot be blank, using stdout only")
+	}
+
+	return lc
+}
+
+// NewClientStdOut creates an instance of LoggingClient that expects to log to stdout and does not check logTarget
+func NewClientStdOut(owningServiceName string, isRemote bool, logLevel string) LoggingClient {
+	if !IsValidLogLevel(logLevel) {
+		logLevel = models.InfoLog
+	}
+
```

Comment:
Code is same in NewClient.  Both instances should be extracted into the common newClient().